default.config.yml

```diff
@@ -4,8 +4,7 @@
 # ansible  #
 ############
 
-ansible_become_method: sudo 
-ansible_become_flags: "-E"
+ansible_become_flags: "-H -S -n --preserve-env=HOMEBREW_CACHE"
```

Comment:
`-E` pass all env variables, is a security issue.

Here, we pass only the required env variable.

`-H -S -n` is the default value of ansible, `-S` is for the password